NoSoft-SA/nspack
# feature/load_defaults

CHANGELOG entry:

### Added
- Added Load and Order defaults using new pattern
### Changed
- Fixed create new load form order

How to test this code via the UI:
for new fg_loads, on create of load some values should be defaulted.
for orders when creating a new load the consignee and billing_client are now set to order customer
